packages/lib/src/instrumentation.ts

```diff
@@ -14,6 +14,8 @@ import {
 import { buildInfo, BuildInfo, recordBuildInfo } from "./buildInfo";
 import { HISTOGRAM_NAME } from "./constants";
 
+let IS_EAGERLY_PUSHED = false;
+let _pushMetrics = () => {};
```

Comment:
What's up with the underscore?

Reply:
benno knows — i love underscores

another personal convention for something that can be reassigned across multiple functions, will remove